packages/admin/admin-rte/src/core/Controls/LinkControls.tsx

```diff
@@ -1,44 +1,55 @@
-import { ButtonGroup, ComponentsOverrides, Theme } from "@mui/material";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import { ThemedComponentBaseProps } from "@comet/admin";
+import { ButtonGroup, ComponentsOverrides, css, styled, Theme, useThemeProps } from "@mui/material";
 import * as React from "react";
 
 import LinkToolbarButton from "../extension/Link/ToolbarButton";
 import LinksRemoveToolbarButton from "../extension/LinksRemove/ToolbarButton";
 import { IControlProps } from "../types";
 
-function LinkControls(p: IControlProps & WithStyles<typeof styles>) {
+export interface RteLinkControlsProps extends IControlProps, ThemedComponentBaseProps<{ root: typeof ButtonGroup; item: "div" }> {}
+
+function StyledLinkControls(inProps: RteLinkControlsProps) {
     const {
         options: { supports: supportedThings, overwriteLinkButton, overwriteLinksRemoveButton },
-        classes,
-    } = p;
+        slotProps,
+        ...restProps
+    } = useThemeProps({ props: inProps, name: "CometAdminRteLinkControls" });
 
     const LinkButtonComponent = overwriteLinkButton ? overwriteLinkButton : LinkToolbarButton;
     const LinksRemoveButtonComponent = overwriteLinksRemoveButton ? overwriteLinksRemoveButton : LinksRemoveToolbarButton;
 
     return (
-        <ButtonGroup classes={{ root: classes.root }}>
-            <div className={classes.item}>{supportedThings.includes("link") && <LinkButtonComponent {...p} />}</div>
-            <div className={classes.item}>{supportedThings.includes("links-remove") && <LinksRemoveButtonComponent {...p} />}</div>
-        </ButtonGroup>
+        <Root {...slotProps?.root} {...restProps}>
+            <Item {...slotProps?.item}>{supportedThings.includes("link") && <LinkButtonComponent {...inProps} />}</Item>
+            <Item {...slotProps?.item}>{supportedThings.includes("links-remove") && <LinksRemoveButtonComponent {...inProps} />}</Item>
```

Comment:
Previously the props defined in the theme for this component would be included in the components props (now `inProps`) and would therefore be included in `<Item {...props} />` this is now no longer the case, since we need to get that manually with `useThemeProps()`. 
Since you are now passing in `inProps` directly, the props defined in the theme are no longer included, but they should be. 

You can do this similarly as before, by not destructuring the props directly, but later in another line:

```tsx
const props = useThemeProps(inProps, ...);
const { options: {...}, slotProps, ...restProps } = props;

// ...

<Item {...props} />
<Item {...props} />
```